lib/jsonapi_compliable.rb

```diff
@@ -31,13 +35,19 @@
 require "jsonapi_compliable/util/sideload"
 require "jsonapi_compliable/util/hooks"
 
+require 'jsonapi_compliable/adapters/null'
+
 # require correct jsonapi-rb before extensions
 if defined?(Rails)
   require 'jsonapi_compliable/rails'
 else
   require 'jsonapi/serializable'
 end
 
+if defined?(ActiveRecord)
```

Comment:
Would the `ActiveSupport.on_load(:active_record)` hook would work here?

Reply:
We do that too ([see the railtie](https://github.com/richmolj/jsonapi_compliable/blob/1.0-attribute-fire/lib/jsonapi_compliable/railtie.rb#L4-L10)). This is a "nice to have" to support ActiveRecord in a non-Rails context.